common/src/main/java/codes/atomys/advr/screens/AdvancementReloadedScreen.java

```diff
@@ -537,7 +538,7 @@ public void renderBackground(final GuiGraphics context, final int mouseX, final
         break;
       case Configuration.BackgroundStyle.ACHIEVEMENT:
         this.selectedTab.ifPresent(tab -> {
-          final ResourceLocation textureResourceLocation = tab.getDisplay().getBackground()
+          final ResourceLocation textureResourceLocation = tab.getDisplay().getBackground().map(ClientAsset::texturePath)
```

Comment:
`texturePath` aren't valid because `context.blit` transform to path internally :dumb:

![image](https://github.com/user-attachments/assets/9ac30e55-1057-411b-a39b-bb2322b9d658)

So I have switch it to `id`